### ISSUE TITLE
Add the ability to make baton persistant

### ISSUE THREAD

**Issue by skluthe** [joshuathompson/baton#12]
Would really like to have there be a command to run like baton console that would keep it persistent so that I could have easy keyboard shortcuts to change my music. Not sure how feasible that would be but it's a feature I would really enjoy.

**Comment by joshuathompson** [comment]
Yeah, it's a good idea.  When I have some time I will check it out.  What commands would you want @skluthe?

I'm thinking...
- Show current song
- Next / Previous
- Volume
- Toggle shuffle
- Pause/Play

**Comment by skluthe** [comment]
Those are definitely great. I was thinking a way to navigate your playlists
would be amazing. Similar to sconsify but something that actually works.
